docker/docsite/Dockerfile

```diff
@@ -19,8 +19,11 @@ ARG USER
 ARG UID
 ARG GID
 
-RUN apt update -qqy \
-    && apt install -qqy --no-install-recommends \
+ENV DEBIAN_FRONTEND=noninteractive
+ENV DEBCONF_NOWARNINGS=yes
+
+RUN apt-get update -qqy \
```

Comment:
There should be no need to change from `apt install` to `apt-get install` in here and in the rest of the changes. `apt` does work for these builds currently.